### ISSUE TITLE
Logrotate fails on Salt logs from RPM packages

### ISSUE THREAD

**Issue by highlyunavailable** [saltstack/salt#19588]
```
# /usr/sbin/logrotate /etc/logrotate.conf --debug
Ignoring salt because of bad file mode.

# ls /etc/logrotate.d -l
-rwxr-xr-x.  1 root root  452 Nov  7 21:08 salt

# chmod -x /etc/logrotate.d/salt

# /usr/sbin/logrotate /etc/logrotate.conf --debug
reading config file salt
```

Tested on fedora 21, but it may affect other RPM packages. Package is `salt.noarch 2014.7.0-3.fc21 @updates-testing`

I think the spec file (https://github.com/saltstack/salt/blob/develop/pkg/rpm/salt.spec#L171) just needs `-m 0644` added to the line with the logrotate config file, but I'm not at all familiar with RPM build specs.

If you want, I can submit a pull request with this change if that's what you think the correct action is.

**Comment by terminalmage** [comment]
Thanks for reporting. I am the RedHat packager, and I'll make sure to modify the spec to set the permissions to 0644.

**Comment by terminalmage** [comment]
This should be fixed for the upcoming bugfix release, 2014.7.1.

**Comment by highlyunavailable** [comment]
Awesome, thank you.
